microservices/time-series-analytics/docs/user-guide/get-started.md

```diff
@@ -46,7 +46,7 @@
 ## Clone source code
 
 ```bash
-git clone https://github.com/open-edge-platform/edge-ai-libraries.git
+git clone https://github.com/open-edge-platform/edge-ai-libraries.git -b release-1.2.0
```

Comment:
Thanks @ajagadi1 for creating this PR.
I don't think we have to port these changes as these are specific docs changes which should reside only at release-1.2.0 branch.

If you are OK, I can undo these changes in this branch.